src/main/resources/static/eiffel-rabbit-mq-post-receive-hook.soy

```diff
@@ -15,25 +15,43 @@
     {param value: $config['rabbit.url'] /}
     {param descriptionText: 'RabbitMQ server address to connect to.' /}
```

Comment:
Please expand help message. Give an example what is expected from user - should address contain http:// or any other protocol?